tests/extrusion/test_steady_advection_3D_extr.py

```diff
@@ -127,7 +127,7 @@ def test_near_to_far(mesh, DG1, W):
     out = Function(DG1)
     solve(a == L, out)
 
-    assert max(abs(out.dat.data - inflow.dat.data)) < 3.5e-7
+    assert max(abs(out.dat.data - inflow.dat.data)) < 3.9e-7
```

Comment:
Likewise these numbers do not have to change if you port the default solver tolerance.